pmm-app/src/shared/components/helpers/Humanization.ts

```diff
@@ -73,8 +73,10 @@ export const humanize = {
       case 'percent':
         if (metricValue !== 0 && metricValue < 0.0001) {
           res = '<0.01%';
-        } else {
+        } else if (metricValue % 1 === 0) {
```

Comment:
Where the tests updated for this case? I think `Number.isInteger` could be useful here!

Reply:
Thanks, that check is better